cypress/support/generate.js

```diff
@@ -0,0 +1,12 @@
+import { faker } from '@faker-js/faker';
+
+function generateUser() {
+  const randomNumber = Math.random().toString().slice(2, 6);
```

Comment:
Consider adding a comment here explaining the purpose of the `randomNumber` and why it's used in the `username`. This will improve the readability of your code.